File: desktop/src/ir/doorbash/hexy/desktop/DesktopLauncher.java
package ir.doorbash.hexy.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

import ir.doorbash.hexy.HexyGame;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.width = 360;
		config.height = 600;
		new LwjglApplication(new HexyGame(), config);
	}
}

File: desktop/src/ir/doorbash/hexy/desktop/DesktopLauncher.kt
package ir.doorbash.hexy.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import ir.doorbash.hexy.HexyGame

object DesktopLauncher {
    @JvmStatic
    fun main(vararg args: String) {
        val config = LwjglApplicationConfiguration()
        config.width = 800
        config.height = 480
        LwjglApplication(HexyGame(), config)
    }
}